support-e2e/tests/tieredCheckout.test.ts

```diff
@@ -102,3 +108,49 @@ test.describe('Subscribe/Contribute via the Tiered checkout)', () => {
 		});
 	});
 });
+
+test.describe('Subscribe (S+) incl PromoCode via the Tiered checkout', () => {
+	testDetailsPromo.forEach((testDetails) => {
+		test(`${testDetails.frequency} (S+) Subscription incl PromoCode at Tier-2 with Credit/Debit card - UK`, async ({
+			context,
+			baseURL,
+		}) => {
+			const page = await context.newPage();
+			const testFirstName = firstName();
+			const testLastName = lastName();
+			const testEmail = email();
+			await setupPage(
+				page,
+				context,
+				baseURL,
+				`/uk/contribute?promoCode=PLAYWRIGHT_TEST_SPLUS`,
+			);
+			await page.getByRole('tab').getByText(testDetails.frequency).click();
+			await expect(
+				page
+					.getByText(
+						` ${
+							testDetails.frequency === `Monthly`
+								? `month for the first 3 months`
+								: `year for the first year`
+						}, then`,
+					)
+					.first(),
+			).toBeVisible();
+			await page
+				.locator(
+					`:nth-match(button:has-text("Subscribe"), ${testDetails.tier})`,
+				)
+				.click();
+			await setTestUserDetails(page, testFirstName, testLastName, testEmail);
+			await page.getByRole('radio', { name: 'Credit/Debit card' }).check();
+			await fillInCardDetails(page);
+			await checkRecaptcha(page);
+			var paymentButtonRegex = new RegExp(
+				'Pay £([0-9]+|(([0-9]+).([0-9]+))) per (year|month)',
+			);
+			await page.getByText(paymentButtonRegex).click();
+			await expect(page).toHaveURL(`/uk/thankyou`, { timeout: 600000 });
```

Comment:
Do we want to be matching the amount in the discounted details?

We could use something like
```ts
const monthlyRegex = /£([\d])\/month for the first 3 months, then/;
const promoDescription = page.getByText(monthlyRegEx);
const amount = promoDescription.textContent().match(monthlyRegex)[1];

// Or something like this?
```

see: https://playwright.dev/docs/api/class-locator#locator-text-content

Reply:
As the original tests do not do this perhaps we should shift this to a seprate Playwright PR? I could not get the `match` to work and think may need a little more time

Reply:
Super, let's get some playwright testing time in